src/cairo_interaction.rs

```diff
@@ -68,9 +68,8 @@ impl ContextExt for Context {
         }
     }
 
-    fn cairo_surface_create_from_pixbuf<'a, P: Into<Option<&'a Window>>>(pixbuf: &Pixbuf, scale: i32, for_window: P) -> Option<Surface> {
+    fn cairo_surface_create_from_pixbuf(pixbuf: &Pixbuf, scale: i32, for_window: Option<&Window>) -> Option<Surface> {
```

Comment:
@sdroege Not related to this PR, you don't know why `gdk::Window` don't used with `IsA` ?

Reply:
That's a good question actually.

Reply:
Oh, this not generated code, so it related to this PR actually 😉

Reply:
Should be `IsA`